### DISCUSSION TITLE
Redirect to user's available page

### DISCUSSION THREAD

**Discussion by IldusGaf** [refinedev/refine#4845] [Q&A]
Good afternoon! I want to make a redirect to an accessible user page after authorization. For example, user1 directs to the "/posts" page and user2 directs to the "/users" page. 
I tried in the login method in AuthProvider in param "redirectTo" 
`return {
        success: true,
        redirectTo,
      };`
Redirect happens, but immediately redirect to the first element in resource array.

**Comment by BatuhanW** [comment]
Hey @IldusGaf your issue is probably related to https://github.com/refinedev/refine/issues/4782

Could you try to remove `fallback` prop in Authenticated component and see if it fixes your issue?

**Reply by IldusGaf** [reply]
Hi @BatuhanW, thanks for your attention.
Remove `fallback` prop in Authenticated and don't return `
{
        authenticated: true,
      };
`
in check method in authProvider gives behavior change.
1. Add search param `?to=%2FredirectTo`.
2. Re-login redirect to the `redirectTo` page, but it does not work (white page).

It seems that the problem is related to [#4782](https://github.com/refinedev/refine/issues/4782).
